app/javascript/helpers/forms/CreateInvitationHelpers.jsx

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+
+export const validationSchema = yup.object({
+  emails:
+    yup.string()
+      .required('Please enter your email.')
+      .email('Entered value does not match email format.'),
+});
+
+export const createInvitaionFormFields = {
+  emails: {
+    label: 'Emails',
```

Comment:
This is irrelevant to this PR but I think we'd need to refactor the forms templates to support `i18n` and to improve it,
Here the labels will not be localized,
We can discuss during meetings,